rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/export/catalog/ExportToCatalogConfirmWindow.java

```diff
@@ -183,50 +194,50 @@ private void fillBucketList() {
             request.send();
         } catch (RequestException e) {
             String errorMessage = "Request sent to catalog failed";
-            GWT.log(errorMessage, e);
+            LogModel.getInstance().logCriticalMessage(errorMessage);
             this.windowLabel.setContents(errorMessage);
         }
     }
 
-    private void requestNodeSourceInBucket(String bucketName) {
+    private void requestCatalogObjectsAndRevisions(String bucketName) {
         if (!bucketName.equals(SELECT_A_BUCKET_OPTION)) {
-            CatalogRequestBuilder catalogRequestBuilder = new CatalogRequestBuilder(null);
-            catalogRequestBuilder.requestNodeSourcesForBucket(bucketName, new RequestCallback() {
+            CatalogRequestBuilder catalogRequestBuilder = new CatalogRequestBuilder();
+            catalogRequestBuilder.requestCatalogObjects(bucketName, this.kind, new RequestCallback() {
                 @Override
                 public void onResponseReceived(Request request, Response response) {
-                    JSONArray nodeSources = JSONParser.parseStrict(response.getText()).isArray();
-                    nodeSourceRevised = false;
+                    JSONArray catalogObjects = JSONParser.parseStrict(response.getText()).isArray();
+                    catalogObjectRevised = false;
                     revisionLabel.setContents("(Initial commit)");
-                    for (int i = 0; i < nodeSources.size(); i++) {
-                        JSONObject nodeSource = nodeSources.get(i).isObject();
-                        String nodeSourceNameInBucket = nodeSource.get("name").isString().stringValue();
-                        if (nodeSourceNameInBucket.equals(ExportToCatalogConfirmWindow.this.nodeSourceName)) {
-                            nodeSourceRevised = true;
+                    for (int i = 0; i < catalogObjects.size(); i++) {
+                        JSONObject catalogObject = catalogObjects.get(i).isObject();
+                        String catalogObjectName = catalogObject.get("name").isString().stringValue();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "catalogObjectName" which hides the field declared at line 80. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)